_pages/en_US/installing-boot9strap-(fredtool).txt

```diff
@@ -27,17 +26,32 @@ As an additional safety measure, we will perform an additional check for custom
 1. Hold the (Select) button
 1. Power on your device while still holding the (Select) button
 1. If the check was successful, you will boot to the HOME Menu and you may proceed with this guide
+1. Power off your device
 
-If you see a configuration menu, you already have CFW, and continuing with these instructions may BRICK your device! Follow [Checking for CFW](checking-for-cfw) to upgrade your existing CFW.
+If you see a configuration menu, or the console immediately powers off, you already have CFW, and continuing with these instructions may BRICK your device! Follow [Checking for CFW](checking-for-cfw) to upgrade your existing CFW.
```

Comment:
the comma between menu and or is unnecessary

Reply:
Resolved.